shap/cext/_cext_gpu.cc

```diff
@@ -0,0 +1,188 @@
+#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
```

Comment:
Did you run valgrind memory leak checks (or something similar) on the GPU version?

Reply:
The GPUTreeShap repo is heavily tested, I haven't run anything specific to its integration here. I can look into it.